src/HH4b/corrections/data/ttbar_sfs/glopart-v2/ttbarcorr_2022.json

```diff
@@ -0,0 +1,427 @@
+{
+  "schema_version": 2,
+  "description": "ttbar corrections for 2022",
+  "corrections": [
+    {
+      "name": "ttbar_corr_Tau3OverTau2_2022",
+      "description": "ttbar correction Tau3OverTau2 for 2022",
+      "version": 1,
+      "inputs": [
+        {
+          "name": "Tau3OverTau2",
```

Comment:
The `Tau3OverTau2` SFs should be the same (or at least similar) in the old file and the new file, correct?

Reply:
Yes, I confirm that the `Tau3OverTau2` SFs agree within error bars.